resources/public/api/conf/2.0/other_retrieve.yaml

```
@@ -6,30 +6,12 @@ description: |
   A National Insurance number and tax year must be provided.
   
   ###  Test data
-  <p>Scenario simulations using Gov-Test-Scenario headers is only available in the sandbox environment.</p>
-    <table>
-      <thead>
-        <tr>
-          <th>Header Value (Gov-Test-Scenario)</th>
-          <th>Scenario</th>
-        </tr>
-      </thead>
-    <tbody>
-      <tr>
-        <td><p>N/A - DEFAULT</p></td>
-        <td><p>Simulates success response.</p></td>
-      </tr>
-      <tr>
-        <td><p>DYNAMIC</p></td>
-        <td><p>The following response values will change to correspond to the values submitted in the request:
-          <ul>
-              <li>taxYear</li>
-              <li>submittedOn</li>
-          </ul>
-        </p></td>
-      </tr>
-    </tbody>
-  </table>
+  Scenario simulation using Gov-Test-Scenario headers is only available in the sandbox environment.
+  
+  | Header Value (Gov-Test-Scenario) | Scenario                                                                                                                        |
+  |----------------------------------|---------------------------------------------------------------------------------------------------------------------------------|
+  | N/A - DEFAULT                    | Simulates success response.                                                                                                     |
```

```suggestion
    | N/A - DEFAULT                    | Simulates success response. |
```
Can we make the column's length shorter? Similar to the other comments.